src/vos/vos_io.c

```diff
@@ -1659,6 +1661,71 @@ akey_update_recx(daos_handle_t toh, uint32_t pm_ver, daos_recx_t *recx,
 	return rc;
 }
 
+int
+vos_key_flags_set(struct umem_instance *umm, struct vos_krec_df *krec)
```

Comment:
This function name looks bit confusing to me, it's trying to mark the item "aggregate-able" but not for setting a common feat flag, so vos_key_mark_agg() or something else looks better to me.

Reply:
Yeah, i can fix that